pkg/terraform/exec/plugins/vsphereprivate/config.go

```diff
@@ -60,6 +66,13 @@ func (cw *ConfigWrapper) Client() (*VSphereClient, error) {
 	if err != nil {
 		return nil, err
 	}
+	ctx, cancel := context.WithTimeout(context.Background(), defaultAPITimeout)
```

Comment:
> https://golang.org/pkg/context/#TODO
```
func Background 1.7

func Background() Context

Background returns a non-nil, empty Context. It is never canceled, has no values, and has no deadline. It is typically used by the main function, initialization, and tests, and as the top-level Context for incoming requests.
func TODO 1.7

func TODO() Context

TODO returns a non-nil, empty Context. Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter). 
```
i think TODO fits better here.

Reply:
ok...will update it